x-pack/legacy/plugins/maps/server/maps_telemetry/maps_telemetry.test.js

```diff
@@ -15,10 +15,10 @@ describe('buildMapsTelemetry', () => {
   test('returns zeroed telemetry data when there are no saved objects',
     async () => {
 
-      const gisMaps = [];
-      const result = buildMapsTelemetry(gisMaps, settings);
+      const result = buildMapsTelemetry([], [], settings);
```

Comment:
Can you keep the named parameters. What is the first `[]`, what is the second `[]`? Would be nice to know this without having to reference  `buildMapsTelemetry` signature.